### ISSUE TITLE
Error while installing on OSX

### ISSUE THREAD

**Issue by maltitco** [node-red/node-red#3614]
### Current Behavior

Unable to install on OSX:
sudo npm install -g --unsafe-perm node-red
returns:
npm WARN deprecated axios@0.27.0: Formdata complete broken, incorrect build size

changed 301 packages, and audited 302 packages in 9s

31 packages are looking for funding
  run `npm fund` for details

3 high severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: newest
- Node.js version: 16.15.0
- npm version: 8.10.0
- Platform/OS: OSX 12.3.1
- Browser:

**Comment by hardillb** [comment]
Nothing in what you've posted actually shows an error, just a warning.

Node-RED should be installed and it should have added `node-red` to the path.

**Comment by maltitco** [comment]
@hardillb Yes it is true, it is just a warning and node-red starts normally.
